test/agentchat/contrib/test_postgresql_agent.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3 -m pytest
+
+import os
+import sys
+
+import pytest
+
+from autogen import config_list_from_json
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
+from conftest import skip_openai  # noqa: E402
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
this line too.

Reply:
Will do, but I'm going to need to change the following line too:

from `from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402`
to `from agentchat.test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402`